### PR TITLE
Feature/#88 merge sorted array

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

**Review by ryujumoto — COMMENTED** [review]
Can you solve this with more efficient solution?

**Comment by Rwatana** [comment]
I solved the problem by merging.

File: 88_merge_sorted_array.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from typing import List
```
Comment (ryujumoto) [inline 88_merge_sorted_array.py:4]: Write time complexity and space one.

**Comment by Rwatana** [comment]
I revised. Please check my code.

File: 88_merge_sorted_array.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# time: (n)
+# space: (n)
```
Comment (ryujumoto) [inline 88_merge_sorted_array.py]: time: O(m+n)
space: O(1)

**Review by ryujumoto — COMMENTED** [review]
Check my review.

**Comment by Rwatana** [comment]
I revised. Please check.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**